patches/tModLoader/Terraria.Localization.Content/es-ES.tModLoader.json

```diff
@@ -154,7 +154,7 @@
 		"MBUpdateAll": "Actualizar todos",
 		"MBReloadBrowser": "Recargar Navegador",
 		"MBGettingData": "Obteniendo datos...",
-		"MBPopulatingBrowser": "Rellenando navegador...",
+		"MBPopulatingBrowser": "Completando navegador...",
```

Comment:
Are you sure 'Completing browser' is the right translation? That's a bit of an odd one..

Reply:
Mmm maybe that's right. Yeah, leave Rellenando. 
I thought that the second one was a lil better yesterday, as it was a synonim, but now thinking on what does it refers to, uh, I'm not sure.
Sorry about that.

Reply:
If you revert it, I can accept the PR. Rest looks fine

Reply:
Okey, done.